src/libraries/System.ServiceProcess.ServiceController/README.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceProcess.ServiceController
+Provides the [`System.ServiceProcess.ServiceController`](https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.servicecontroller) class, which allows you to connect to a Windows service, manipulate it, or get information about it.
+
+Note that `System.ServiceProcess.ServiceController` supports Windows only.
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.serviceprocess.
+
+## Contribution Bar
+- [ ] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
```

Comment:
Does this mean we wouldn't consider new features or we would (wondering whether I'd expect an X in the box).
We believe this library is mature but it's not deprecated and it's how we recommend to create Windows services so I'd assume we'd accept improvements (?)